apps/feedback/lib/message.ex

```diff
@@ -89,7 +92,10 @@ defmodule Feedback.Message do
           no_request_response: boolean,
           incident_date_time: DateTime.t(),
           photos: [Plug.Upload.t()] | nil,
-          ticket_number: String.t() | nil
+          ticket_number: String.t() | nil,
+          mode: String.t(),
+          line: String.t(),
+          vehicle: String.t()
```

Comment:
Since these both have no default value and are optional, I think we want to include `nil` in here too.